code/UI/Player/PossessionInfo/PossessionInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Sandbox;
+using Sandbox.UI;
+
+namespace TTT.UI;
+
+[UseTemplate]
+public class PossessionInfo : Panel
```

Comment:
tbh I'd be fine with naming this PunchMeter or something like that, what do you think?